Dockerfile.minio

```diff
@@ -0,0 +1,5 @@
+FROM minio/minio:RELEASE.2021-07-27T02-40-15Z
+EXPOSE 9000
+EXPOSE 9001
+COPY ./data/* /data/
+CMD [ "minio", "server", "--console-address", ":9001", "/data"]
```

Comment:
I _think_ that this can just be in the docker-compose.build-override.yaml file. Instead of doing the COPY, we just do the volume and set the rest of the options there. There are options in docker-compose for exposing ports and setting the cmd.